docs/nightly/en/db-cloud-shared/migrate/migrate-from-influxdb.md

```diff
@@ -0,0 +1,366 @@
+...
+```
+
+### Export Data from InfluxDB v2 Server
+
+Create a temporary directory to store the exported data of InfluxDB.
+
+```shell
+mkdir -p /path/to/export
+```
+
+Use the [`influx inspect export-lp` command](https://docs.influxdata.com/influxdb/v2/reference/cli/influxd/inspect/export-lp/) of InfluxDB to export data in the bucket to line protocol.
+
+```shell
+influxd inspect export-lp \
+  --bucket-id <bucket-id> \
+  --engine-path /var/lib/influxdb2/engine/ \
+  --end <end-time> \
+  --output-path /path/to/export/data
+```
+
+- The `--bucket-id` flag specifies the bucket ID to be exported.
+- The `--engine-path` flag specifies the path to the engine directory, as configured in the [InfluxDB data settings](https://docs.influxdata.com/influxdb/v2.0/reference/config-options/#engine-path).
+- The `--end` flag specifies the end time of the data to be exported. Must be in [RFC3339 format](https://datatracker.ietf.org/doc/html/rfc3339), such as `2024-01-01T00:00:00Z`.
+You can use the timestamp when simultaneously writing data to both GreptimeDB and InfluxDB as the end time.
+- The `--output-path` flag specifies the output directory.
+
+The outputs look like the following:
+
+```json
+{"level":"info","ts":1714377321.4795408,"caller":"export_lp/export_lp.go:219","msg":"exporting TSM files","tsm_dir":"/var/lib/influxdb2/engine/data/307013e61d514f3c","file_count":1}
+{"level":"info","ts":1714377321.4940555,"caller":"export_lp/export_lp.go:315","msg":"exporting WAL files","wal_dir":"/var/lib/influxdb2/engine/wal/307013e61d514f3c","file_count":1}
+{"level":"info","ts":1714377321.4941633,"caller":"export_lp/export_lp.go:204","msg":"export complete"}
+```
+
+The exported data in InfluxDB line protocol looks like the following:
+
+```txt
+cpu,cpu=cpu-total,host=bogon usage_idle=80.4448912910468 1714376180000000000
+cpu,cpu=cpu-total,host=bogon usage_idle=78.50167052182304 1714376190000000000
+cpu,cpu=cpu-total,host=bogon usage_iowait=0 1714375700000000000
+cpu,cpu=cpu-total,host=bogon usage_iowait=0 1714375710000000000
+...
+```
+
+### Import Data to GreptimeDB
+
+Before importing data to GreptimeDB, if the data file is too large, it's recommended to split the data file into multiple slices:
+
+```shell
+split -l 100000 -d -a 10 data data.
+# -l [line_count]    Create split files line_count lines in length.
+# -d                 Use a numeric suffix instead of a alphabetic suffix.
+# -a [suffix_length] Use suffix_length letters to form the suffix of the file name.
+```
+
+You can import data using the HTTP API as described in the [write data section](#http-api).
+The Python script provided below will help you in reading data from the files and importing it into GreptimeDB.
+
+Create a Python file named `ingest.py`, ensuring you're using Python version 3.9 or later, and then copy and paste the following code into it.
+
+```python
```

Comment:
I hightly recommend you to use [the script here](https://github.com/GreptimeTeam/demo-scene/tree/main/influxdb-v1-to-greptime#step-3-import-data-to-greptimedb) instead. It's much more concise:

```shell
for file in data/data.*; do
    curl -i -H "Authorization: token $GREPTIME_USERNAME:$GREPTIME_PASSWORD" \
        -X POST "https://${GREPTIME_HOST}/v1/influxdb/write?db=$GREPTIME_DB" \
        --data-binary @${file}
    sleep 1
done
```